addons/account/models/ir_actions_report.py

```diff
@@ -45,3 +48,11 @@ def render_qweb_pdf(self, res_ids=None, data=None):
                     raise UserError(_("Only invoices could be printed."))
 
         return super().render_qweb_pdf(res_ids=res_ids, data=data)
+
+    def _get_rendering_context(self, docids, data):
+        data = data and dict(data) or {}
+        ids = isinstance(docids, int) and [docids] or docids
+        invoice = self.env['account.move'].search([('id', 'in', ids)])
+        rounding = invoice and min(invoice.mapped('currency_id.rounding')) or 0.0
+        data.update({'float_compare': float_compare, 'rounding': rounding or self.env.user.company_id.currency_id.rounding})
```

Comment:
one key per line please

Reply:
ok

Reply:
You shouldn't be passing `rounding` anyway as you should take it from the currency set on the invoice.
If there is no currency, it is the one from the currency set on the company set on the invoice. Not the one from the env.